modules/social_features/social_group/src/Plugin/views/field/SocialGroupViewsBulkOperationsBulkForm.php

```diff
@@ -70,7 +70,7 @@ public function viewsForm(array &$form, FormStateInterface $form_state) {
 
     parent::viewsForm($form, $form_state);
 
-    if ($this->view->id() !== 'group_manage_members') {
+    if ($this->view->id() !== 'group_manage_members' || empty($form['output'][0]['#rows'])) {
```

Comment:
This could do with a comment to make it easier to parse. Can probably do this while merging.